requirements/windows.txt

```diff
@@ -1,6 +1,7 @@
 # Windows source distribution requirements
 
 -r zeromq.txt
+-r rabbitmq.txt
```

Comment:
**question** do we always want to install rabbitmq support? I feel like that should be optional :thinking:

Reply:
I agree

Reply:
This was my attempt at making the tests pass in the CICD pipeline and install required dependencies. I've marked tests as "xfail", so this can be rolled back now.